### ISSUE TITLE
Ansible Lint version 26.2.0 run using pre-commit requires python3.14

### ISSUE THREAD

**Issue by chriscroome** [ansible/ansible-lint#4978]
##### Summary

Ansible Lint version `26.2.0` requires `python3.14`.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

I have a `.pre-commit-config.yaml` containing:

```yaml
---
repos:
  # https://github.com/ansible/ansible-lint/releases
  - repo: https://github.com/ansible/ansible-lint.git
    rev: v26.2.0
    hooks:
      - id: ansible-lint
        name: Ansible Lint
        language: python
        args:
          - 'all.yml'
        additional_dependencies:
          # https://pypi.org/project/ansible/
          - ansible==13.3.0
          # https://github.com/kellyjonbrazil/jc/releases
          - jc==1.25.6
          # https://github.com/jmespath/jmespath.py/tags
          - jmespath==1.0.1
```

This fails with:

```
[INFO] Initializing environment for https://github.com/ansible/ansible-lint.git.
[INFO] Initializing environment for https://github.com/ansible/ansible-lint.git:ansible==13.3.0,jc==1.25.6,jmespath==1.0.1.
[INFO] Installing environment for https://github.com/ansible/ansible-lint.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/python3', '-mvirtualenv', '/home/chris/.cache/pre-commit/repojol6vgb7/py_env-python3.14', '-p', 'python3.14')
return code: 1
stdout:
    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.14'
stderr: (none)
Check the log at /home/chris/.cache/pre-commit/pre-commit.log
```
```console (paste below)
ansible-lint --version
```

This is on Debian Trixie which has:

```
python3 --version
Python 3.13.5
```

##### Desired Behavior

Would it be possible for `ansible-lint` to work with more than one version of Python?

See also [this issue](https://github.com/ansible/ansible-lint/issues/4946).

**Comment by chriscroome** [comment]
I've created a [thread on the forums about this issue](https://forum.ansible.com/t/runnng-ansible-lint-via-pre-commit-requires-python3-14/45493).

**Comment by saz** [comment]
Set `language_version` in your `.pre-commit-config.yaml` to `python3`:
```
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index 6aed17a..9f109c7 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -37,3 +37,4 @@ repos:
     hooks:
       - id: ansible-lint
         args: [--fix]
+        language_version: python3
```

**Comment by chriscroome** [comment]
Thanks @saz that solves it!
